.github/workflows/workspace.yaml

```
@@ -1,37 +1,37 @@
 name: Workspace
```

Let's rename this workflow to be clearer. Maybe even rename the file.
```suggestion
name: Lint
```

---

renamed workflow and file